### ISSUE TITLE
Getting some weird bugs after trying to update bootimage

### ISSUE THREAD

**Issue by steveklabnik** [rust-osdev/bootimage#10]
So, I'm working on updating intermezzos to the new version of bootimage, however I'm getting some failures on CI and locally that are strange. The only change I've made is to add the `build` subcommand to `bootimage.

Here's the CI failure: https://travis-ci.org/intermezzOS/kernel/jobs/370152249#L521

```text
error: current package believes it's in a workspace when it's not:
current:   /home/travis/build/intermezzOS/kernel/target/intermezzos/debug/bootloader/Cargo.toml
workspace: /home/travis/build/intermezzOS/kernel/Cargo.toml
this may be fixable by ensuring that this crate is depended on by the workspace root: /home/travis/build/intermezzOS/kernel/Cargo.toml
thread 'main' panicked at 'Bootloader download failed.', src/build.rs:250:5
```

and locally:

```text
> bootimage build --target=intermezzos
Building kernel
   Compiling rlibc v1.0.0
   Compiling vga v0.1.0 (file:///C:/Users/steve/src/intermezzos/kernel/vga)
error[E0463]: can't find crate for `compiler_builtins`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: Could not compile `rlibc`.
warning: build failed, waiting for other jobs to finish...
error[E0463]: can't find crate for `compiler_builtins`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: Could not compile `vga`.

To learn more, run the command again with --verbose.
```

Any ideas on what's going on here? I'm using bootimage 0.2.6 and `rustc 1.27.0-nightly (ac3c2288f 2018-04-18)`.

**Comment by phil-opp** [comment]
Should be fixed in https://github.com/rust-osdev/bootimage/commit/df6c73c44bb102eeb8b7f4f4d4583895400165a7. Published as version 0.2.7. I also verified that intermezzos builds again.

**Comment by steveklabnik** [comment]
Thank you! As always, you're the best <3
